examples/jsm/loaders/ifc/web-ifc-api.js

```diff
@@ -0,0 +1,4792 @@
+          process["exit"](status);
+        };
+        Module["inspect"] = function() {
+          return "[Emscripten Module object]";
+        };
+      } else if (ENVIRONMENT_IS_SHELL) {
+        if (typeof read != "undefined") {
+          read_ = function shell_read(f) {
+            return read(f);
+          };
+        }
+        readBinary = function readBinary2(f) {
+          var data;
+          if (typeof readbuffer === "function") {
+            return new Uint8Array(readbuffer(f));
+          }
+          data = read(f, "binary");
+          assert(typeof data === "object");
+          return data;
+        };
+        if (typeof scriptArgs != "undefined") {
+          arguments_ = scriptArgs;
+        } else if (typeof arguments != "undefined") {
+          arguments_ = arguments;
+        }
+        if (typeof quit === "function") {
+          quit_ = function(status) {
+            quit(status);
+          };
+        }
+        if (typeof print !== "undefined") {
+          if (typeof console === "undefined")
+            console = {};
+          console.log = print;
+          console.warn = console.error = typeof printErr !== "undefined" ? printErr : print;
+        }
+      } else if (ENVIRONMENT_IS_WEB || ENVIRONMENT_IS_WORKER) {
+        if (ENVIRONMENT_IS_WORKER) {
+          scriptDirectory = self.location.href;
+        } else if (typeof document !== "undefined" && document.currentScript) {
+          scriptDirectory = document.currentScript.src;
+        }
+        if (_scriptDir) {
+          scriptDirectory = _scriptDir;
+        }
+        if (scriptDirectory.indexOf("blob:") !== 0) {
+          scriptDirectory = scriptDirectory.substr(0, scriptDirectory.lastIndexOf("/") + 1);
+        } else {
+          scriptDirectory = "";
+        }
+        {
+          read_ = function shell_read(url) {
+            var xhr = new XMLHttpRequest();
+            xhr.open("GET", url, false);
+            xhr.send(null);
+            return xhr.responseText;
+          };
+          if (ENVIRONMENT_IS_WORKER) {
+            readBinary = function readBinary2(url) {
+              var xhr = new XMLHttpRequest();
+              xhr.open("GET", url, false);
+              xhr.responseType = "arraybuffer";
+              xhr.send(null);
+              return new Uint8Array(xhr.response);
+            };
+          }
+          readAsync = function readAsync2(url, onload, onerror) {
+            var xhr = new XMLHttpRequest();
+            xhr.open("GET", url, true);
+            xhr.responseType = "arraybuffer";
+            xhr.onload = function xhr_onload() {
+              if (xhr.status == 200 || xhr.status == 0 && xhr.response) {
+                onload(xhr.response);
+                return;
+              }
+              onerror();
+            };
+            xhr.onerror = onerror;
+            xhr.send(null);
+          };
+        }
+        setWindowTitle = function(title) {
+          document.title = title;
+        };
```

Comment:
I think this method is unused, but setting a document title as part of a library isn't a good idea imho.

Reply:
Hey, I think this is part of the code that Emscripten generates automatically. We'll take a look to see if we can do something about it.